project/AppDependencies.scala

```diff
@@ -22,68 +6,68 @@ object AppDependencies {
   val play26Version = "2.6.20"
   val play27Version = "2.7.4"
 
-  val compile: Seq[ModuleID] =
-    PlayCrossCompilation.dependencies(
-      shared = Seq(
-        "com.typesafe" % "config"    % "1.3.3",
-        "org.slf4j"    % "slf4j-api" % "1.7.25",
-        // empty http-core added to force eviction 
-        // as classes from this lib have been inlined in http-verbs
-        "uk.gov.hmrc" %% "http-core"          % "2.2.0",
-        // force dependencies due to security flaws found in jackson-databind < 2.9.x using XRay
-        "com.fasterxml.jackson.core"     % "jackson-core"            % "2.9.8",
-        "com.fasterxml.jackson.core"     % "jackson-databind"        % "2.9.8",
-        "com.fasterxml.jackson.core"     % "jackson-annotations"     % "2.9.8",
-        "com.fasterxml.jackson.datatype" % "jackson-datatype-jdk8"   % "2.9.8",
-        "com.fasterxml.jackson.datatype" % "jackson-datatype-jsr310" % "2.9.8"
-      ),
-      play25 = Seq(
-        "com.typesafe.play" %% "play-json" % play25Version,
-        "com.typesafe.play" %% "play-ws"   % play25Version,
-        // force dependencies due to security flaws found in xercesImpl 2.11.0
-        // only applies to play 2.5 since it was removed from play 2.6
-        // https://github.com/playframework/playframework/blob/master/documentation/manual/releases/release26/migration26/Migration26.md#xercesimpl-removal
-        "xerces" % "xercesImpl" % "2.12.0",
-        // empty http-verbs-play-25 added to force eviction
-        // as classes from this lib have been inlined in http-verbs
-        "uk.gov.hmrc" %% "http-verbs-play-25" % "1.0.0"
-      ),
-      play26 = Seq(
-        "com.typesafe.play" %% "play-json" % "2.6.9",
-        "com.typesafe.play" %% "play-ws"   % play26Version
-      ),
-      play27 = Seq(
-        "com.typesafe.play" %% "play-json" % play27Version,
-        "com.typesafe.play" %% "play-ws"   % play27Version
-      )
-    )
+  val compileCommon = Seq(
+    "com.typesafe" % "config"    % "1.3.3",
+    "org.slf4j"    % "slf4j-api" % "1.7.25",
+    // empty http-core added to force eviction
+    // as classes from this lib have been inlined in http-verbs
+    "uk.gov.hmrc" %% "http-core"          % "2.2.0",
+    // force dependencies due to security flaws found in jackson-databind < 2.9.x using XRay
```

Comment:
This might be redundant now as pointed out by Chris on one of my PRs, I updated to: https://github.com/hmrc/logback-json-logger/blob/master/build.sbt#L35-L36

Reply:
Does that mean that all 5 of the jackson imports can be replaced with just those 2?

Reply:
I believe the others are pulled in transitively, but best way to know for sure is to remove them and see if everything still resolves ok

Reply:
Change made, pushed.